frontend/src/pages/analytics/metadata/user-attributes/MetadataUserAttributesTable.tsx

```diff
@@ -245,72 +242,38 @@ const MetadataUserAttributes: React.FC = () => {
   };
 
   return (
-    <AppLayout
-      toolsHide
-      content={
-        <MetadataTable
-          resourceName="UserAttribute"
-          tableColumnDefinitions={COLUMN_DEFINITIONS}
-          tableContentDisplay={CONTENT_DISPLAY}
-          tableFilteringProperties={FILTERING_PROPERTIES}
-          tableI18nStrings={{
-            loadingText:
-              t('analytics:metadata.labels.tableLoading') || 'Loading',
-            emptyText: t('analytics:metadata.labels.tableEmpty'),
-            headerTitle: t('analytics:metadata.userAttribute.title'),
-            headerRefreshButtonText: t('common:button.refreshMetadata'),
-            filteringAriaLabel: t(
-              'analytics:metadata.userAttribute.filteringAriaLabel'
-            ),
-            filteringPlaceholder: t(
-              'analytics:metadata.userAttribute.filteringPlaceholder'
-            ),
-            groupPropertiesText: t('button.groupPropertiesText'),
-            operatorsText: t('button.operatorsText'),
-            clearFiltersText: t('button.clearFiltersText'),
-            applyActionText: t('button.applyActionText'),
-            useText: t('common:table.useText'),
-            matchText: t('common:table.matchText'),
-            matchesText: t('common:table.matchesText'),
-          }}
-          loadHelpPanelContent={() => {
-            console.log(1);
-          }}
-          setShowDetails={(
-            show: boolean,
-            data?:
-              | IMetadataEvent
-              | IMetadataEventParameter
-              | IMetadataUserAttribute
-          ) => {
-            setShowSplit(show);
-            if (data) {
-              setCurUserAttribute(data as IMetadataUserAttribute);
-            }
-          }}
-          fetchDataFunc={listMetadataUserAttributes}
-          fetchUpdateFunc={updateMetadataUserAttributeInfo}
-        ></MetadataTable>
-      }
-      headerSelector="#header"
-      navigation={
-        <Navigation
-          activeHref={`/analytics/${projectId}/app/${appId}/metadata/user-attributes`}
-        />
-      }
-      splitPanelOpen={showSplit}
-      onSplitPanelToggle={(e) => {
-        setShowSplit(e.detail.open);
+    <MetadataTable
+      resourceName="UserAttribute"
+      tableColumnDefinitions={COLUMN_DEFINITIONS}
+      tableContentDisplay={CONTENT_DISPLAY}
+      tableFilteringProperties={FILTERING_PROPERTIES}
+      tableI18nStrings={{
```

Comment:
I see the similar I18N String above, can we define these as a common constant?

Reply:
The problem is that can not use i18n in constant file.